Modules/Scorm2004/data/seq_editor/seqtemplates.sql.php

```diff
@@ -1,47 +1,48 @@
 <?php
 
-$ilDB->query("CREATE TABLE sahs_sc13_seq_condition(`condition` varchar(50), `seqNodeId` int(11), `measureThreshold` varchar(50), `operator` varchar(50), `referencedObjective` varchar(50) );");
+$ilDB->query("CREATE TABLE sahs_sc13_seq_condition(" . $ilDB->quoteIdentifier('condition') . " varchar(50), " . $ilDB->quoteIdentifier('seqNodeId') . " int(11), " . $ilDB->quoteIdentifier('measureThreshold') . " varchar(50), " . $ilDB->quoteIdentifier('operator') . " varchar(50), " . $ilDB->quoteIdentifier('referencedObjective') . " varchar(50) );");
```

Comment:
@Uwe-Kohnle what is this file about? could you please move this to proper db-update-steps if really needed (I did not found usages of this file in trunk).